ApplyToTheIndustry/Assets/Scripts/Jobs/JobPosting.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/* CLASS: JobPosting
+ * Scriptable Object used mainly for storing
+ * all information on any individual job posting
+ */
+[CreateAssetMenu(menuName = "ScriptableObjects/JobPosting")]
+public class JobPosting : ScriptableObject
+{
+    // Data that the player sees
+    public string companyName;
```

Comment:
Depends on what company info will be needed but I agree it may play into stat calculations at some point. In which case however, we can easily swap the two.